osx/cookbooks/android/default.rb

```diff
@@ -0,0 +1,6 @@
+require 'itamae/plugin/resource/cask'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.